sway-lib-std/src/revert.sw

```diff
@@ -5,22 +5,22 @@ use ::logging::log;
 use ::error_signals::FAILED_REQUIRE_SIGNAL;
 
 /// Will either panic or revert with a given number depending on the context.
-/// If used in a predicate, it will panic.
-/// If used in a contract, it will revert.
 ///
-/// ### Arguments
+/// # Additional Information
 ///
-/// * `code` - The code with which to revert the program.
+/// If used in a predicate, it will panic.
+/// If used in a contract, it will revert.
 ///
-/// ### Reverts
+/// # Arguments
 ///
-/// Reverts when called in a contract.
+/// * `code`: [u64] - The code with which to revert the program.
 ///
-/// ### Panics
+/// # Panics
 ///
-/// Panics when called in a predicate.
+/// * Panics when called in a contract.
```

Comment:
Since this is already mentioned in the additional information section, maybe "Panics unconditionally" would be better here?